src/Service/DataSynchronizationService.php

```diff
@@ -271,6 +274,26 @@ public function syncIssuesForProject(int $projectId, ?callable $progressCallback
                     }
                 }
 
+                $epicArray = explode(',', $issueDatum->epicName);
```

Comment:
Instead of exploding here, issueDatum should include an array of Epics, and LeantimeDataProvider and JiraDataProvider should supply the array of epics.